xml/System.Runtime.InteropServices/UnmanagedCallersOnlyAttribute.xml

```diff
@@ -132,7 +132,15 @@ Supplied types must be from the official <xref:System.Runtime.CompilerServices>
       </ReturnValue>
       <Docs>
         <summary>Optional. If omitted, no named export is emitted during compilation.</summary>
-        <remarks>To be added.</remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The export name must be a valid identifier for a native symbol on the target platform. The identifier must not conflict with system provided native identifiers (such as: names of methods in the C standard library, or POSIX/Win32 API names).
```

Comment:
If I change the DNNE sample to export `free` instead of `FancyName` (the exact thing the customer did in the linked bug), the compilation fails with

```
main.c
..\bin\Debug\net6.0\win-x64\Sample_native.h(24): warning C4028: formal parameter 1 different from declaration
..\bin\Debug\net6.0\win-x64\Sample_native.h(24): error C2371: 'free': redefinition; different basic types
C:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt\corecrt_malloc.h(89): note: see declaration of 'free'
```

Are you sure you want to deal with this in DNNE or can we keep the broad definition?

Reply:
> I think keeping the above statement is appropriate.

Wouldn't the same thing happen if the user hijacks any method that DNNE calls into during its operation? The problem feels the same. On Windows I see DNNE builds libraries statically linked with the C runtime - what if I hijack a Win32 API used to implement the C runtime? And if it's linked dynamically, what if I hijack whatever API is used from the subset of CRT that is still linked into the executable?

Reply:
> Wouldn't the same thing happen if the user hijacks any method that DNNE calls into during its operation?

I don't know the answer to that. I do see your point though. Let's go with your current phrasing, we can always clarify in the future. Thanks for pushing back.